### PR TITLE
Visually center the main content

### LINKED ISSUE #415: Center the main view
Main view is not visually centered. The main view should always be centered regardless of the nav menu.

### PR DESCRIPTION
**Pull request by samuelli**
Fixes #415.

### CODE REVIEW

File: src/client/public/styles/nav.css [hunk 1]
```diff
@@ -1,11 +1,16 @@
...
   flex-direction: column;
   margin-right: var(--grid-size);
   font-size: 14px;
   line-height: 20px;
   flex-shrink: 0;
+  position: absolute;
+
+  --grid-size: 16px;
+  --max-desktop-width: 1024px;
+  --nav-size: 250px;
```
Comment (gauntface) [inline src/client/public/styles/nav.css]: Stupid question - will these variables be picked up even though they are defined *after* usage?

File: src/client/public/styles/nav.css [hunk 2]
```diff
@@ -1,11 +1,16 @@
 nav-element {
-  width: 250px;
+  width: var(--nav-size);
```
Comment (gauntface) [inline src/client/public/styles/nav.css]: --nav-size is defined twice, once in the root and once in the element, would it make sense to use params with a hardcoded fallback? `width: var(--nav-size, 250px);`

File: src/client/public/styles/nav.css (hunk 1 above)
Comment (samuelli) [reply src/client/public/styles/nav.css]: Removed, shouldn't be there.

File: src/client/public/styles/nav.css (hunk 2 above)
Comment (samuelli) [reply src/client/public/styles/nav.css]: Fixed.

**Pull request merged**